agent/fw_drupal8.c

```diff
@@ -322,12 +321,57 @@ NR_PHP_WRAPPER_END
  * function call of this type gets to name the txn.
  */
 NR_PHP_WRAPPER(nr_drupal8_name_the_wt_cached) {
-  const char* name = "page_cache";
+  char* name = NULL;
   zval** retval_ptr = NR_GET_RETURN_VALUE_PTR;
+  zval* request = NULL;
+
+  zend_function* nrfn = NULL;
+
+  zval* controller = NULL;
 
   (void)wraprec;
 
   NR_PHP_WRAPPER_REQUIRE_FRAMEWORK(NR_FW_DRUPAL8);
+
+  request = nr_php_arg_get(1, NR_EXECUTE_ORIG_ARGS);
+
+  if (0 == nr_php_is_zval_valid_object(request)) {
+    nrl_verbosedebug(
+        NRL_TXN, "Drupal 8 PageCache::get does not have a request parameter");
+
+    goto end;
+  }
+
+  if (false
```

Comment:
Small nit - but both `nr_php_is_zval_valid_object` and `nr_php_object_instanceof_class` return a value of "0" (not `false`) so can both of these checks use the same value for consistency?

Reply:
da14399002566dc6741a84065b8f96b407f9064b